incubator/monochart/requirements.yaml

```diff
@@ -0,0 +1,4 @@
+dependencies:
+- name: common
+  version: 0.0.1
```

Comment:
latest version is `0.0.3`

Reply:
Should I just deprecate it and rewrite all stuff without its magic?
Due to https://github.com/cloudposse/helm-chart-scaffolding/issues/8